function-aws/features/test-aws-junit/skeleton/src/test/java/@defaultPackage.path@/@project.className@FunctionTest.java

```diff
@@ -1,20 +1,19 @@
 package @defaultPackage@;
 
-import io.micronaut.context.ApplicationContext;
-import io.micronaut.runtime.server.EmbeddedServer;
-import org.junit.Test;
+import io.micronaut.test.annotation.MicronautTest;
+import org.junit.jupiter.api.Test;
+import javax.inject.Inject;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Maybe this should be `*` instead of should `assertEquals`

Reply:
I'm not a big fan of `*` imports. If `assertEquals` is the only method used in the generated tests, there is no reason to import them all. From there, users can use any other methods, and their IDE will do the import on demand, changing to `*` when necessary.